### ISSUE TITLE
Sawada Standard Design Comment on WCAG 2.1 WD

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag21#733]
W3C AG ワーキンググループ 様（Cc：植木様）

SAWADA STANDARD DESIGN の澤田と申します。
はじめまして。

WCAG 2.1 ワーキングドラフトについてコメントいたします。
（文字化けしていたらごめんなさい）

・達成基準 2.6.1 動作による起動［レベル A］について

レベル A であるなら、「かつ」以降の無効化に関する条件は必須にしなくても良いのではないかと思います。
無効化できなくてもユーザインタフェース コンポーネントで操作ができれば、目的が達成できる可能性は高いと思いますので。

・達成基準 1.3.4 共通した（or 一般的な）目的の特定［レベル AA］について

WCAG 2.0 において「ユーザインタフェース コンポーネント」はフォーム要素だけではなくリンクも含むことになっていますので、「全ての」ユーザインタフェース コンポーネントの目的をプログラムによる解釈可能にすることは、レベル AA としてハードルが高すぎると感じます。

以上です。
よろしくお願いいたします。

***
SAWADA STANDARD DESIGN

**Comment by awkawk** [comment]
Issue submitted after the comment deadline.

**Comment by MakotoUeki** [comment]
https://lists.w3.org/Archives/Public/public-agwg-comments/2018Jan/0002.html

**English translation**

**About SC 2.6.1 Motion Actuation (Level A)**
If this SC is at Level A, the phrase "and can be disabled to prevent accidental actuation" can be removed. Even if it cannot be disabled, users will be able to complete their tasks when they can interact with the UI component.

**About SC 1.3.4 1.3.4 Identify Common Purpose (Level AA)**
In WCAG 2.0, "user interface component" includes not only form controls but also links. It will result in a higher hurdle at Level AA to require "all" user interface components to be programmatically determined.

**Comment by awkawk** [comment]
(Proposed response - not official, for WG review)
Thanks you for the comments.

For 2.6.1, the reason that the phrase "can be disabled to prevent accidental actuation" is needed is that the accidental actuation can interfere with the user's ability to successfully operate the functionality via the UI components. As a result, it isn't sufficient to just provide a way to use functionality via the UI components as some users can still be adversely affected - for example, a user with hand tremors might continually be shaking a device and activating a feature unintentionally. That user is likely to want to activate that feature at times, and needs the UI component-based operating pathway to do so successfully, but also needs to have the sensing of motion stopped in order to continue to use the site or application.

For 1.3.4, the version of this SC in the CR version of WCAG 2.1 doesn't use "user interface component". Instead it is focused on input fields: "The meaning of each input field collecting information about the user can be programmatically determined when" so links will not be covered by this SC.

**Comment by awkawk** [comment]
The WG decided on the above response, so we changed the text in the comment containing the proposed response to read "[Official WG Response]". Please confirm is you are satisfied with the response within 1 week. If we haven't heard a response in a week we will regard the resolution as satisfactory.

**Comment by SawadaStdDesign** [comment]
Thank you for reply. I understand.
